### PR TITLE
feat(Bonus Pagamenti Digitali): [#176091990] Improve credit card error screen UI

### PR DESCRIPTION
**Pull request by Undermaken**
## Short description
This PR improves onboarding credit card errors screens

| add wallet error  | verification error | wallets load error |
| ------------- | ------------- | ------------- |
add wallet error | verification error| verification error

### CODE REVIEW

**Comment by thisisjp** [comment]
@Undermaken could we adjust the margins and title width of the first two screens in order to match the third one? :)

**Comment by Undermaken** [comment]
> @Undermaken could we adjust the margins and title width of the first two screens in order to match the third one? :)

Sure. Since that component is used widely on several screens could we do this task on a dedicated story to ensure that it won't create side effects?

File: ts/screens/wallet/ConfirmCardDetailsScreen.tsx [hunk 1]
```diff
@@ -134,17 +142,44 @@ class ConfirmCardDetailsScreen extends React.Component<Props, State> {
...
-          isInPayment
-            ? I18n.t("wallet.saveCardInPayment.header")
-            : I18n.t("wallet.saveCard.header")
-        }
-        contextualHelpMarkdown={contextualHelpMarkdown}
-        faqCategories={["wallet_methods"]}
-      >
+    // shown when wallets pot is in error state
+    const walletsInErrorContent = (
+      <SafeAreaView style={bonusVacanzeStyle.flex}>
```
Comment (CrisTofani) [inline ts/screens/wallet/ConfirmCardDetailsScreen.tsx:147]: Maybe it's better to use `IOStyles.flex` which is a global style value, the one you used is more feature scoped, what do you think?

**Review by CrisTofani — CHANGES_REQUESTED** [review]

File: ts/screens/wallet/ConfirmCardDetailsScreen.tsx (hunk 1 above)
Comment (Undermaken) [reply ts/screens/wallet/ConfirmCardDetailsScreen.tsx]: update within f891368

**Review by CrisTofani — APPROVED** [review]

**Pull request merged**